ui/src/components/FeedMewWrapper.vue

```diff
@@ -92,4 +98,27 @@ const hideProfile = (index: number) => {
 const keepShowingProfile = (index: number) => {
   window.clearTimeout(profileHideTimeouts.value[index]);
 };
+const toggleLickMew = async () => {
+   // check if already likes
+   if(isLickedByMe()) {
+      // unlick
+      await unlickMew(props.mew.mewEntryHash);
+      // refreshMew();
+   } 
+   else {
+      // lick
+      await lickMew(props.mew.mewEntryHash);
+      // refreshMew();
+   }
+};
+
+const isLickedByMe = () => {
+  // check if my agent key is in the returned list of likes
+  return props.mew.likes.includes(myAgentPubKey);
+};
+
+// const refreshMew = async () => {
```

Comment:
Why is this commented out?

Reply:
I was hoping to refresh a singular mew in the mews feed after reacting to it (to update the counts) but it was more difficult than I was anticipating. So I commented out the function for now and copied the pattern of refreshing the entire feed on certain events being emitted from child components. The comments have been removed.